codegen/lib/graphql_java_gen/templates/APISchema.java.erb

```diff
@@ -201,8 +205,8 @@ public class <%= schema_name %> {
                 <% end %>
 
                 <% if type_names_set.include?('Node') %>
-                  public List<Node> getNodes() {
-                    List<Node> children = new ArrayList<>();
+                  public List<com.shopify.graphql.support.Node> getNodes() {
```

Comment:
oh right, because there will be another one defined in the file generated by this template